docs/getting-started.md

```diff
@@ -442,47 +444,40 @@ automatically save the edits to files that you make.**
 ### Add an `age` column
 
 1. The "Explorer" on the left hand side lists the files and folders in
-   your research repository. Find and click on the `study_definition.py`
-   file inside the `analysis` folder. This file specifies the population
-   that you'd like to study (dataset rows) and what you need to know
-   about them (dataset columns).
+   your research repository. Find and click on the `dataset_definition.py`
+   file inside the `analysis` folder. This file contains a dataset definition,
+   specifying the population that you'd like to study (dataset rows)
+   and what you need to know about them (dataset columns).
+   It is written in [ehrQL](/ehrql/).
```

Comment:
suggestion:

```suggestion
   It is written in [ehrQL](/ehrql/); OpenSAFELY's electronic health record query language.
```

I think it's the first time ehrQL is referred to, so worth clarifying for the reader.